lib/domain/usecase/search/search_interactor.dart

```diff
@@ -4,12 +4,12 @@ import 'package:fluffychat/di/global/get_it_initializer.dart';
 import 'package:fluffychat/domain/app_state/search/search_interactor_state.dart';
 import 'package:fluffychat/domain/model/contact/contact_query.dart';
 import 'package:fluffychat/domain/model/extensions/contact/contact_extension.dart';
-import 'package:fluffychat/domain/model/extensions/contact/presentation_contact_list_extension.dart';
+import 'package:fluffychat/presentation/extensions/contact/presentation_contact_list_extension.dart';
```

Comment:
I still see a lot of things in `presentation` here. It means that `domain` still know about `presentation`